snapfaas/src/syscalls.proto

```diff
@@ -194,6 +194,17 @@ message FSCreateFacetedDir {
   string name = 2;
 }
 
+message InvokeService {
+  repeated PathComponent serv = 1;
+  optional string route = 2;
```

Comment:
I don't think letting the caller arbitrarily add to the path of the service via this `route` parameter is reasonable given the rest of the design.